G/Givaro/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Givaro"
+version = v"4.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/linbox-team/givaro.git", "fc6cac7820539c900dde332326c71461ba7b910b"),
```

Comment:
This is not [version 4.2.0](https://github.com/linbox-team/givaro/releases/tag/v4.2.0) (released Dec 14, 2021), but rather https://github.com/linbox-team/givaro/commit/fc6cac7820539c900dde332326c71461ba7b910b (a commit from March 2024), the current master branch.

Is that really necessary? If it is, then I think at the very least this should be pointed out in a comment. E.g.
```suggestion
    # The following points to a snapshot of the Givaro branch from March 2024
    # instead of 4.2.0 (the last release) because ... TODO INSERT REASON
    GitSource("https://github.com/linbox-team/givaro.git", "fc6cac7820539c900dde332326c71461ba7b910b"),
```

Of course ideally the givaro folks would be convinced to just tag a new release (seems overdue anyway) so that here a tarball can be referenced instead of a git commit. However based on the open PRs there I guess this might be hard, so I would not want to require this here.